src/go/src/ebw/www/repoHandlers.go

```diff
@@ -193,10 +193,15 @@ func pullRequestView(c *Context) error {
 		return err
 	}
 
```

Comment:
Good to check for errors here, but if you just return err, we get a fairly opaque error returning up the stack. I recommend logging the error at this point, before returning it. Use `ebw/util` and `return util.Error(err)`.